modyn/config/schema/pipeline/evaluation/metric.py

```diff
@@ -11,7 +11,7 @@
 class Metric(ModynBaseModel):
     name: str = Field(description="The name of the evaluation metric.")
     config: dict[str, Any] = Field({}, description="Configuration for the evaluation metric.")
-    evaluation_transformer_function: str = Field(
+    evaluation_transformer_function: str | None = Field(
         "",
```

Comment:
There's a bit of inconsistency with this now I think, at some point we defaulted to None, sometimes to "". Anyways what's important is that we can also instantiate metrics without this (e.g. top n accuracy :D)

Reply:
Good to know, thanks for the pointer!